abahachuk/ptushki-backend
# fix: EPMCUXDA-7299 Combine importing xls endpoints on back-end

### What does this PR do?

- Change existing functionality with combining 2 endpoints for import observation.
If I upload excel file with observations and no errors occurred then all row are valid and they we be uploaded.
- In case of errors errors will be sent without any data import
 
### How should I test this?
- You can download template file and fill it in with data
- Send request to `/observations/import/xls` with filled template file
- You will receive response (See screenshots below from postman)

### Jira ticket related to this PR

- [EPMCUXDA-7299](https://jira.epam.com/jira/browse/EPMCUXDA-7299)

#### Type of change

- [ ] Adds new features.
- [x] Changes existing functionality.
- [ ] Deprecates features.
- [ ] Removes features.
- [ ] Fixes any bug.
- [ ] Other: security, build process, infrastructure, tests, docs, etc.

### Screenshots (if appropriate)

Success import:
![image](https://user-images.githubusercontent.com/19889121/70601139-78c65e80-1c02-11ea-830f-11009ecb5a14.png)

Import if errors:
![image](https://user-images.githubusercontent.com/19889121/70601350-e70b2100-1c02-11ea-80f8-07b456d10543.png)


### Questions (if any)

`If exists, to another developer or reviewer.`
